src/ui/AccountDropdown/index.svelte

```diff
@@ -38,41 +39,58 @@
         <a
           class="btn-ghost"
           href="https://app.santiment.net/alerts?tab=1"
-          on:click={window.__onLinkClick}>My alerts</a
-        >
+          on:click={window.__onLinkClick}>My alerts</a>
         <a class="btn-ghost" href="https://app.santiment.net/assets" on:click={window.__onLinkClick}
-          >My watchlists</a
-        >
+          >My watchlists</a>
         <a
           class="btn-ghost"
           href="https://insights.santiment.net/my"
-          on:click={window.__onLinkClick}>My insights</a
-        >
+          on:click={window.__onLinkClick}>My insights</a>
         <a
           href="https://insights.santiment.net/new"
           class="write btn-1 btn--s"
-          on:click={window.__onLinkClick}>Write insight</a
-        >
+          on:click={window.__onLinkClick}>Write insight</a>
       </section>
       <hr />
     {/if}
 
     <section>
+      {#if !currentUser}
```

Comment:
Create new `section` in the `:else` branch of the `{#if currentUser}` statement on the LOC 33